apache/ozone
# HDDS-11154. Improve Overview Page UI

## What changes were proposed in this pull request?
This PR will add a new Overview page which contains a better implementation for the UI

Please describe your PR in detail:
* This PR adds a new Overview Page with improved layout and design.
* The new page would help to eventually improve the overall look and feel for Recon
* Currently the page is disabled to the user so as to not affect any existing functionality
* Apart from this it also contains the initial setup for a few components that will be used in the future PR as a part of the improvement effort.

## What is the link to the Apache JIRA
https://issues.apache.org/jira/browse/HDDS-11154

## How was this patch tested?
This patch was tested manually via both localhost and building and running Ozone via docker
Screenshot:
<img width="1726" alt="Screenshot 2024-08-05 at 12 10 29" src="https://github.com/user-attachments/assets/84a8ced3-1d8d-4927-9cb9-9c90b350c803">

### How to test this new UI?
Currently the UI is disabled and hidden via using a switch so that users are not able to access it accidentally.
In order to switch to the new UI please follow the below steps:
- Remove the [disable property](https://github.com/apache/ozone/blob/f98f8c6ac35d52e3bc28e235eeef00c948c42972/hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.tsx#L68) from the AntDSwitch component in app.tsx
- (For testing after build) Make sure to build after removal of the disable property in the above step
- Click on the switch in the top right corner of the UI
- This should take you to the new page
**Please Note**: Currently the links for the cards point to blank pages as those have not been implemented yet
